src/microbe_stage/PatchMap.cs

```diff
@@ -445,7 +431,6 @@ private void RecreateAdjacencies()
         {
             var patch1 = Patches[id1];
             var patch2 = Patches[id2];
-
```

Comment:
Please leave this be, I prefer extra spacing, in fact I added either this or the one below this myself.